sql/_13_issues/_23_1h/answers/cbrd_24598.answer

```diff
@@ -0,0 +1,32 @@
+===================================================
+0
+===================================================
+decode( ?:0 , '', c, null, c, -1)    
+-1     
+
+===================================================
+decode( ?:0 , '', c, null, c, -1)    
+-1     
+
+===================================================
+decode( ?:0 , '', c, null, c, -1)    
+-1     
+
+===================================================
+0
+===================================================
+decode( ?:0 , '', c, null, c, 'Z')
+Z
+
+===================================================
+decode( ?:0 , '', c, null, c, 'Z')
+Z
```

Comment:
An error occurs and it fails.
Shouldn't the error code be included in the answer?